android/versioned-abis/expoview-abi39_0_0/src/main/java/abi39_0_0/expo/modules/splashscreen/SplashScreenModule.kt

```diff
@@ -9,6 +9,15 @@ import abi39_0_0.org.unimodules.core.errors.CurrentActivityNotFoundException
 import abi39_0_0.org.unimodules.core.interfaces.ActivityProvider
 import abi39_0_0.org.unimodules.core.interfaces.ExpoMethod
 
+// Below import is added explicitly to provide a redirection from versioned code realm to unversioned code realm.
```

Comment:
Personally I think the long version of this explanation (current one) belongs to this pull request's description while in the code we could put like a one-two lines summary pointing out that it's intentional (if anything, this pattern isn't uncommon in the codebase). _Other solutions_ section contains information that can become outdated easily.